servicetalk-http-router-jersey-internal/src/main/java/io/servicetalk/http/router/jersey/internal/BufferPublisherInputStream.java

```diff
@@ -147,12 +147,11 @@ public static <T> T handleEntityStream(final InputStream entityStream,
         return inputStreamHandler.apply(wrappedStream, allocator);
     }
 
-    @Nullable
     private static byte[] getBytes(final Buffer content) {
         final int readableBytes = content.readableBytes();
 
         if (readableBytes == 0) {
-            return null;
+            return EMPTY_ARRAY;
```

Comment:
This is an interesting change. It seems right to me, but how did it survive this long? Maybe it's just never called with an empty buffer?

Reply:
yes, looks like that, glad it was caught